--- /usr/lib/python3.8/site-packages/hyperkitty/views/mailman.py.orig
+++ /usr/lib/python3.8/site-packages/hyperkitty/views/mailman.py
@@ -53,7 +53,7 @@
                 msg = "Missing setting: %s" % attr
                 logger.error(msg)
                 raise ImproperlyConfigured(msg)
-        if (request.META.get("REMOTE_ADDR") not in
+        if ('0.0.0.0' not in settings.MAILMAN_ARCHIVER_FROM) and (request.META.get("REMOTE_ADDR") not in
                 settings.MAILMAN_ARCHIVER_FROM):
             logger.error(
                 "Access to the archiving API endpoint was forbidden from "
