--- /usr/lib/python3.8/site-packages/hyperkitty/views/mailman.py.orig
+++ /usr/lib/python3.8/site-packages/hyperkitty/views/mailman.py
@@ -48,12 +48,12 @@
 def key_and_ip_auth(func):
     @wraps(func)
     def _decorator(request, *args, **kwargs):
-        for attr in ('MAILMAN_ARCHIVER_KEY', 'MAILMAN_ARCHIVER_FROM'):
+        for attr in ('MAILMAN_ARCHIVER_KEY', 'MAILMAN_ARCHIVER_FROM', 'MAILMAN_ARCHIVER_IP_CHECK'):
             if not hasattr(settings, attr):
                 msg = "Missing setting: %s" % attr
                 logger.error(msg)
                 raise ImproperlyConfigured(msg)
-        if (request.META.get("REMOTE_ADDR") not in
+        if (settings.MAILMAN_ARCHIVER_IP_CHECK == 'true') and (request.META.get("REMOTE_ADDR") not in
                 settings.MAILMAN_ARCHIVER_FROM):
             logger.error(
                 "Access to the archiving API endpoint was forbidden from "
